Gallery.Api/Services/ArticleService.cs

```diff
@@ -162,6 +166,8 @@ public async Task<ViewModels.Article> CreateAsync(ViewModels.Article article, Ca
                     // UserArticles
                     await _userArticleService.LoadUserArticlesAsync(teamArticle.Id, ct);
                 }
+                var verb = new Uri("https://w3id.org/xapi/dod-isd/verbs/created");

```

Comment:
All of the Verb and ActivityType hard-coded strings should be added to appsettings and XApiOptions, so that they can be configured.